lua/lazyvim/plugins/extras/editor/fzf.lua

```diff
@@ -42,18 +42,31 @@ return {
       config.defaults.actions.files["ctrl-t"] = require("trouble.sources.fzf").actions.open
 
       -- Toggle root dir / cwd
-      config.defaults.actions.files["alt-c"] = function(_, ctx)
+      config.defaults.actions.files["ctrl-r"] = function(_, ctx)
         local o = vim.deepcopy(ctx.__call_opts)
         o.root = o.root == false
         o.cwd = nil
         o.buf = ctx.__CTX.bufnr
         LazyVim.pick.open(ctx.__INFO.cmd, o)
       end
+      config.defaults.actions.files["alt-c"] = config.defaults.actions.files["ctrl-r"]
+
+      -- use the same prompt for all
+      local defaults = require("fzf-lua.profiles.default-title")
+      local function fix(t)
+        t.prompt = t.prompt ~= nil and " " or nil
+        for _, v in pairs(t) do
+          if type(v) == "table" then
+            fix(v)
+          end
+        end
+      end
+      fix(defaults)
 
-      return {
-        [1] = "default-title",
+      return vim.tbl_deep_extend("force", opts, defaults, {
```

Comment:
Do we want the defaults to overwrite `opts`? Or the other way around?

Reply:
opts will be empty in this case so it doesn't matter.
User's options are loaded later

Reply:
Yup. Sounds good.